src/openrct2/world/Sprite.cpp

```diff
@@ -736,11 +756,10 @@ void litter_create(int32_t x, int32_t y, int32_t z, int32_t direction, int32_t t
         }
     }
 
-    rct_litter* litter = (rct_litter*)create_sprite(1);
+    rct_litter* litter = (rct_litter*)create_sprite(SPRITE_IDENTIFIER_LITTER);
```

Comment:
`&create_sprite(SPRITE_IDENTIFIER_LITTER)->litter;`

Reply:
What if create_sprite results null? That would dereference 0 trying to access the litter member

Reply:
I don't think that does do a null deref. Otherwise GET_PEEP would null deref.